### PR TITLE
Gom create contenet artifacts

### PR DESCRIPTION
**Pull request by sabaghadar**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/39090

## Description
Add create-content-artifacts support for generic object models
## Screenshots
Paste here any images that will help the reviewer

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/constants.py [hunk 1]
```diff
@@ -1257,6 +1261,10 @@ class ContentItems(Enum):
     LAYOUTS = 'layoutscontainer'
     CLASSIFIERS = 'classifier'
     WIDGETS = 'widget'
+    GENERIC_MODULES = 'genericmodule'
```
Comment (roysagi) [inline demisto_sdk/commands/common/constants.py:1264]: didn't we add these already?

File: demisto_sdk/commands/create_artifacts/content_artifacts_creator.py [hunk 2]
```diff
@@ -296,6 +305,26 @@ def add_widget_as_content_item(self, content_object: ContentObject):
...
+
+    def add_generic_definition_as_content_item(self, content_object: ContentObject):
+        self.content_items[ContentItems.GENERIC_DEFINITIONS].append({
+            'name': content_object.get('name', '')
+        })
+
+    def add_generic_module_as_content_item(self, content_object: ContentObject):
+        self.content_items[ContentItems.GENERIC_MODULES].append({
+            'name': content_object.get('name', '')
+        })
```
Comment (roysagi) [inline demisto_sdk/commands/create_artifacts/content_artifacts_creator.py:326]: is name enough though? Where are the other fields used?

Comment (sabaghadar) [reply demisto_sdk/commands/create_artifacts/content_artifacts_creator.py]: these are the fields in the metadata file of each pack. updated according to- https://github.com/demisto/etc/issues/38101#issuecomment-882368058

Comment (roysagi) [reply demisto_sdk/commands/create_artifacts/content_artifacts_creator.py]: Is that what is displayed in the marketplace? if so let's also add the description there - @Itay4 anything else you can think of?

Comment (sabaghadar) [reply demisto_sdk/commands/create_artifacts/content_artifacts_creator.py]: Right, its what is displayed in the marketplace. description is there- https://github.com/demisto/demisto-sdk/blob/07f5c9b1340adb995ac439e12a406ca74f7af57d/demisto_sdk/commands/create_artifacts/content_artifacts_creator.py#L323-L332

**Review by roysagi — APPROVED** [review]

**Comment by sabaghadar** [comment]
Successfully passed:
content build- https://code.pan.run/xsoar/content/-/pipelines/1462833
private build- https://github.com/demisto/content-private/pull/450
test upload flow -https://app.circleci.com/pipelines/github/demisto/content/103800/workflows/f309587c-7090-47de-83a4-2d920f24e265

**Pull request merged**